scorum/assetlist.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "scorum",
+  "assets": [
+    {
+      "description": "The native token of Scorum",
+      "denom_units": [
+        {
+          "denom": "nscr",
+          "exponent": 0
+        },
+        {
+          "denom": "scr",
+          "exponent": 9
+        }
+      ],
+      "base": "nscr",
+      "name": "Scorum",
+      "display": "scr",
+      "symbol": "SCR",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scr.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scr.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scr.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scr.svg"
+        }
+      ]
+    },
+    {
+      "description": "Scorum Power used for internal mechanics and staking",
+      "denom_units": [
+        {
+          "denom": "nsp",
+          "exponent": 0
+        },
+        {
+          "denom": "sp",
+          "exponent": 9
+        }
+      ],
+      "base": "nsp",
+      "name": "Scorum Power",
+      "display": "sp",
+      "symbol": "SP",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scorum.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scorum.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scorum.png",
```

Comment:
These images do not exist. P{lease upload scorum.png and scorum.svg (unless you mean to point to 'sp.png' and 'sp.svg', in which can please correct this and the image reference in chain.json

Reply:
fixed